types/braintree/index.d.ts

```diff
@@ -976,6 +974,8 @@ declare namespace braintree {
         trialDuration?: number;
         trialDurationUnit?: string;
         trialPeriod?: boolean;
+        id?: string;
```

Comment:
/nit Can we keep the props in alphabetical order? We've done this throughout and I think it makes lookups easier.